.babelrc

```diff
@@ -5,15 +5,20 @@
   "env": {
     "cjs": {
       "presets": [
-        ["es2015", {"loose": true}]
+        ["latest", {"es2015": {"loose": true}}]
       ],
       "plugins": [
         ["add-module-exports"]
       ]
     },
     "es": {
       "presets": [
-        ["es2015", {"loose": true, "modules": false}]
+        ["latest", {"es2015": {"loose": true, "modules": false}}]
+      ]
+    },
+    "test": {
+      "presets": [
+        ["latest", {"es2015": {"loose": true, "modules": "commonjs"}}]
```

Comment:
As discussed above, probably be good to use it instead of babel-preset-latest / es2015 for all builds. Maybe.